docs/source/openshift/index.adoc

```diff
@@ -6,7 +6,7 @@ include::variables.adoc[]
 {project} creates a virtual machine and provisions a local, single-node OpenShift cluster in this VM.
 The following sections describe how {project} can assist you in interacting and configuring your local OpenShift cluster.
 
-For details about managing the {project} VM, see the xref:../using/managing-minishift.adoc#[Managing {project}] section.
+For details about managing the {project} VM, see the xref:../using/basic-commands.adoc#[Managing {project}] section.
```

Comment:
The `xref` here is still named "Managing {project}".

Reply:
ok